sections/adguard-specific/annoyances.txt

```diff
@@ -46,6 +46,7 @@ myonlineradio.hu#$#.js-appAdv { display: none !important; }
 myonlineradio.hu#$#.cover-slider-cont { display: block !important; }
 noizz.hu#$#.leavingPopupLayer { display: none !important; }
 noizz.hu#$#body { overflow: auto !important; }
+bme.hu#?#style:contains(eu-cookie-withdraw-banner)
```

Comment:
You mean the following rule?
```
bme.hu$$style:contains(eu-cookie-withdraw-banner)
```

Reply:
But why would we want to hide a style element? The `$$` (HTML filtering) rule only works on certain platforms and removes the style element from the HTML before it gets loaded into the browser. Similar to uBO's `##^` HTML filtering